src/bash/scripts/eval/set-gcloud-path

```diff
@@ -13,7 +13,7 @@ __am_prompt_set_gcloud_path() {
 		fi
 
 		# add gcloud to the path
-		export PATH="$AM_PROMPT_USER/.gcloud/bin:$PATH"
+		export PATH="$AM_PROMPT_USER/bin/gcloud/bin/:$PATH"
```

Comment:
@patrickserrano Do we want this to be in the prompt user folder. I think we back this folder up every update. I guess there could be advantages but would we have to reinstall each time a user updated prompt?

Reply:
@dmccaffery isn't this why we are installing bins to the $HOME dir so updates do not effect what is installed on the user system

Reply:
@sjk07 it looks like it was changed in the last PR to install to `prompt/user/bin`. 

That said, I could have misinterpreted it given the inconsistent paths.

Reply:
@patrickserrano hmm... we'll have to look into this unless @dmccaffery can give us some insight into the original change. 

This is a blocker for me for releasing v8.3; I don't want to introduce multiple bins of gcloud or other installs when upgrading to the new version of prompt. (unless there is a reason for such a change)

imo upgrading prompt should have nothing to do with other installs on the users system unless they are prompt dependencies (there should be very few). things that are dependent can live in the prompt folder and be backed up incase an upgrade fails.

Lets figure out what we missed in the last updated 😄

Reply:
you merged my WIP -- I was midstream in some things, including this...

Reply:
@dmccaffery my bad, I didn't realize that was still WIP!